src/uct/base/uct_iface.c

```diff
@@ -545,10 +545,10 @@ ucs_config_field_t uct_iface_config_table[] = {
    "UCX_<IB transport>_TX_MIN_INLINE for IB, UCX_MM_FIFO_SIZE for MM",
    UCS_CONFIG_DEPRECATED_FIELD_OFFSET, UCS_CONFIG_TYPE_DEPRECATED},
 
-  {"MAX_BCOPY", "8192",
-   "Maximal size of copy-out sends. The transport is allowed to support any size\n"
-   "up to this limit, the actual size can be lower due to transport constraints.",
-   ucs_offsetof(uct_iface_config_t, max_bcopy), UCS_CONFIG_TYPE_MEMUNITS},
+  {"MAX_BCOPY", "",
+   "The configuration parameter replaced by: "
+   "UCX__SEG_SIZE where is one of: IB, MM, SELF, TCP",
```

Comment:
sorry, looks like github removed '<>' from my comment. it should be:
"UCX_<transport>_SEG_SIZE where <transport> is one of: IB, MM, SELF, TCP""

Reply:
Ok, I see. Thanks! Fixed